### PR TITLE
New article added in the Layer5 Newsroom

### LINKED ISSUE #2389: [News] 6 CNCF Service Mesh Tools
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the [Layer5 Newsroom](https://layer5.io/news).

**Article:**
- Title: 6 CNCF Service Mesh Tools
- Link: :meshery: Meshery and :smp: Service Mesh Performance highlighted as two of six CNCF service mesh tools - https://containerjournal.com/features/6-cncf-service-mesh-tools/

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by MukulKolpe**
**Description**

This PR fixes #2389

**Notes for Reviewers**
Added "6 CNCF Service Mesh Tools" article in the Layer5 Newsroom.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Uh-oh - - https://github.com/layer5io/layer5/runs/4397405029?check_suite_focus=true#step:3:147

File: src/collections/news/2021-10-15-6-cncf-service-mesh-tools/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "6 CNCF Service Mesh Tools"
+subtitle: ""
+date: 2021-10-15 04:00:05 +0000
+author: The Newsroom
+category: Coverage
+thumbnail: ./Service-Mesh-pattern.png
+eurl: https://containerjournal.com/features/6-cncf-service-mesh-tools/#:~:text=5.%20Meshery,of%20Service%20Mesh%20Performance%20(SMP).
```
Comment (leecalcote) [inline src/collections/news/2021-10-15-6-cncf-service-mesh-tools/index.mdx:8]:
```suggestion
eurl: https://containerjournal.com/features/6-cncf-service-mesh-tools/#:~:text=5.%20Meshery,of%20Service%20Mesh%20Performance%20(SMP).
```
```suggestion
eurl: https://containerjournal.com/features/6-cncf-service-mesh-tools
```

File: src/collections/news/2021-10-15-6-cncf-service-mesh-tools/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This is where service mesh comes in. Service mesh helps to apply common observability and security features across applications. It’s typically split into a control plane, used to configure features, and a data plane, consisting of a sidecar proxy alongside each application. Nowadays, several service mesh options exist in the market, each with varying levels of complexity.
+
+The Cloud Native Computing Foundation (CNCF) is home to much of today’s innovative cloud-native technology. The foundation now hosts a few service meshes and related projects. Below, we’ll outline the CNCF service mesh toolset to better understand how engineers can adopt these tools and their benefits.
+
+**Read the full article on [Container Journal](https://containerjournal.com/features/6-cncf-service-mesh-tools/#:~:text=5.%20Meshery,of%20Service%20Mesh%20Performance%20(SMP).).**
```
Comment (leecalcote) [inline src/collections/news/2021-10-15-6-cncf-service-mesh-tools/index.mdx:22]:
```suggestion
**Read the full article on [Container Journal](https://containerjournal.com/features/6-cncf-service-mesh-tools).**
```

**Comment by MukulKolpe** [comment]
Thank you @leecalcote !!

**Pull request merged**